advanced-examples/network-performance/network-performance-iperf3-k8s/workflow.yaml

```diff
@@ -102,30 +109,44 @@ steps:
             imagePullPolicy: Always 
   #TODO Implement signaling once available in the engine
   pcp:
-    plugin: quay.io/arcalot/arcaflow-plugin-pcp:0.2.0
+    plugin:
+      deployment_type: image
+      src: quay.io/arcalot/arcaflow-plugin-pcp:0.8.0
     step: start-pcp
+    stop_if: !expr $.steps.iperf3_client.outputs
     input:
       pmlogger_interval: !expr $.input.pcp_pmlogger_interval
-      run_duration: !expr $.input.time
     deploy:
-      type: kubernetes
+      deployer_name: kubernetes
       connection: !expr $.steps.kubeconfig.outputs.success.connection
       pod:
         metadata:
-          namespace: !expr $.input.k8s_namespace
+          namespace: default
           labels:
             arcaflow: pcp
         spec:
           pluginContainer:
             imagePullPolicy: Always
+              #          affinity:
+              #            podAffinity:
+              #              requiredDuringSchedulingIgnoredDuringExecution:
+              #              - labelSelector:
+              #                  matchExpressions:
+              #                  - key: arcaflow
+              #                    operator: In
+              #                    values:
+              #                    - iperf3-client
+              #                topologyKey: node-role.kubernetes.io/worker
```

Comment:
Are you leaving this commented for a reason, or should it be deleted? (If this is intended "documentation" for a user, then a commented explanation might be a good idea.)

Reply:
Added a note that this is a TODO